Clinical-Genomics/patientMatcher
# Move documentation to dedicated mkdocs pages

### This PR adds | fixes:
- Move documentation to mkdocs (fix #278)
- Documentation autodeploy GitHub action triggered by new release

**How to prepare for test**:
- [ ] On a local conda env, install mkdocs

### How to test:
- Run `mkdocs serve` and make sure there are no errors and all pages are accessible

### Review:

This [version](https://semver.org/) is a:
